tests/test_emulators.py

```diff
@@ -197,8 +250,21 @@ def test_routing(self):
         model.rvp.params = model.params(0.529, -3.396, 407.29, 1.072, 16.9, 0.947)
 
         # TODO: compute this??
+
+        # model.rvp.avg_annual_runoff = function_to_derive_this(area_as_sum_of_all_hrus, q_obs_most_down_stream_means_with_down_sub_id=-1)
```

Comment:
ToDo: Derive average annual runoff based on total basin area and `q_obs` of the most downstream basin (the one with `DownSubId==-1`)